cpp/include/cudf/io/parquet_metadata.hpp

```diff
@@ -46,7 +46,6 @@ enum class TypeKind : int8_t {
   BOOLEAN              = 0,
   INT32                = 1,
   INT64                = 2,
-  INT96                = 3,  // Deprecated
```

Comment:
This should not be removed from the enum as it is still in the official spec.

Reply:
Reverted!